pxtlib/package.ts

```diff
@@ -4,6 +4,35 @@
 /// <reference path="util.ts"/>
 
 namespace pxt {
+
+    export class EditorPackage {
```

Comment:
this does not belong here.

Reply:
I'm not sure of a way to get EditorPackage class from pxtrunner to Package otherwise so if it's not here there can't be an _editorpkg field on Package so pxtrunner has to use any to get this field. I've put it back to where it was, but I had to put the getEditorPackage function back to its original implementation which used any.